### ISSUE TITLE
feat(patch):  Google Feed For Android With No Ads

### ISSUE THREAD

**Issue by AshwinKO** [revanced/revanced-patches#1265]
### Application

Google Android APP

### Issue

is there any way we can revance the google app available on android it shows too many ads and lots of data collection an adblocking  feature which blocks ads in google feed plus search results would be really nice

### Patch

adblocking and data collection blocking, trackers blocking support in google app for android

### Motivation

.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by CnC-Robert** [comment]
That requires root to patch

If you want less data collection use a custom rom

**Comment by oSumAtrIX** [comment]
Why does it require root?

**Comment by AshwinKO** [comment]
> Thats what i guessed as its probably a system app and its a google app so the signature is stored somewhere like youtube

some people have root so can we do it till the time we do it we will keep figuring out how its possible on non root because google app on android has really became a pain to use this days lots of ads have started showing up on feed whatever you search in ecommerce websites starts popping up instantly in google feed with all those assaulting ads

**Comment by oSumAtrIX** [comment]
@CnC-Robert Vanced MicroG is useful for this.

**Comment by CnC-Robert** [comment]
> @CnC-Robert Vanced MicroG is useful for this.

Yeah i just realised this is about the google app, i thought it was the screen you get when swiping left on some launchers

**Comment by AshwinKO** [comment]
> > @CnC-Robert Vanced MicroG is useful for this.
> 
> Yeah i just realised this is about the google app, i thought it was the screen you get when swiping left on some launchers

it is about the google app which includes the g feed i was talking about

**Comment by oSumAtrIX** [comment]
Can you make a screenshot of the ads in question?

**Comment by XBadFellaX** [comment]
> ### Application
> Google Android APP
> 
> ### Issue
> is there any way we can revance the google app available on android it shows too many ads and lots of data collection an adblocking feature which blocks ads in google feed plus search results would be really nice
> 
> ### Patch
> adblocking and data collection blocking, trackers blocking support in google app for android
> 
> ### Motivation
> .
> 
> ### Acknowledgements
> * [x]  I have searched the existing issues and this is a new and no duplicate or related to another open issue.
> * [x]  I have written a short but informative title.
> * [x]  I filled out all of the requested information in this issue properly.

I use adguard and don't see any ads in the google android app. Maybe try that?

**Comment by jobhax25** [comment]
> Can you make a screenshot of the ads in question?

This is what I get.

![Screenshot_20221214_090702_Google](https://user-images.githubusercontent.com/57541289/207632317-2cfe23de-1568-4b14-8e92-7e15b31f1b6d.png)
![Screenshot_20221214_090642_Google](https://user-images.githubusercontent.com/57541289/207632335-5ae4731e-eb56-4461-bdb5-ec385614658a.png)

**Comment by oSumAtrIX** [comment]
I can not see these.

**Comment by AshwinKO** [comment]
> > Can you make a screenshot of the ads in question?
> 
> This is what I get.
> 
> ![Screenshot_20221214_090702_Google](https://user-images.githubusercontent.com/57541289/207632317-2cfe23de-1568-4b14-8e92-7e15b31f1b6d.png) ![Screenshot_20221214_090642_Google](https://user-images.githubusercontent.com/57541289/207632335-5ae4731e-eb56-4461-bdb5-ec385614658a.png)

yeah this is the one look at those sponsored ones its just tonnes of them when you scroll after every 2 article its there

**Comment by shadoxxhd** [comment]
It's getting worse - some ad images penetrate my adguard DNS filter, so it's more and more difficult to distinguish ads from news (you need to scroll past the image to see the small "sponsored" tag).

Even if root is necessary to use this patch, I would greatly appreciate it being available...
